apps/els_lsp/src/els_log_notification.erl

```diff
@@ -0,0 +1,50 @@
+-module(els_log_notification).
+
+-export([start/0]).
+-export([log/2]).
+
+-type lsp_message_type() :: 1 | 2 | 3 | 4.
```

Comment:
It would be good to map the `const`s from the specs to macros and use those instead of the numeric literals. For example:

```
-define(ERROR, 1)
```

And then the following:

```
map_otp_log_level_to_lsp(emergency) -> 1;
```

Would become:

```
map_otp_log_level_to_lsp(emergency) -> ?ERROR;
```